homeassistant/components/ccm15/config_flow.py

```diff
@@ -0,0 +1,76 @@
+"""Config flow for Midea ccm15 AC Controller integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PORT, CONF_SCAN_INTERVAL
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .climate import CCM15Coordinator
+from .const import DEFAULT_INTERVAL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Optional(CONF_PORT, default=80): int,
```

Comment:
Please also add the follwoing import `import homeassistant.helpers.config_validation as cv`

```suggestion
        vol.Optional(CONF_PORT, default=80): cv.port,
```